h/templates/emails/flag_notification.txt.jinja2

```diff
@@ -0,0 +1 @@
+An annotation  has been flagged. Please visit {{ incontext_link }} for more details.
```

Comment:
Tiny typo. Looks like two spaces instead of one after "annotation" @sheetaluk . Same with the HTML template - although that won't make any difference to the rendered output.